src/data/features.ts

```diff
@@ -76,12 +76,12 @@ export const cloudFiles = [
 
 export const previewFiles = [
 	{
-		name: "June 2018",
+		name: "Files v3",
 		icon: "/ui/icons/folder.png",
-		items: 3,
-		added: "Thursday, June 21, 2018 11:24 AM",
-		modified: "Sunday, November 15, 2020 6:50 PM",
-		path: "C:\\Users\\Sara\\Desktop\\June 2018"
+		items: 16,
+		added: "Wednesday, October 6, 2021 42:69 AM",
```

Comment:
While I see that this is quite a funny date, maybe we should stick with a date that would actually work?

Reply:
Or, and hear me out: _what if we don't?_

Reply:
Than we have a sample that is incorrect and can potentially be seen as offensive or incorrect. But thats only my opinion on this, maybe other folks have other opinions here.

Reply:
I don't know, I think 42 and 69 are pretty much considered "meme numbers". If anyone has a problem with it though, I'll definitely change it.